lib/openfoodfacts.dart

```diff
@@ -805,7 +806,7 @@ class OpenFoodAPIClient {
   /// Returns [Status.status] 201 = complete; 400 = wrong inputs + [Status.error]; 500 = server error;
   ///
   /// When creating a [producer account](https://world.pro.openfoodfacts.org/) use [orgName] (former requested_org) to name the Producer or brand
-  static Future<Status> register({
+  static Future<SignUpStatus> register({
```

Comment:
That's a breaking change, isn't it?
I guess you should deprecate this method and create another one. With additional comments about the new feature.

Reply:
Actually the current one will still work, as the SignUpStatus does exactly was Status did before.
So I'm not sure a breaking change is required here

Reply:
Fair enough @g123k: the previous version of the method returned a `Status`, now it returns a `SignUpStatus` which is also a `Status`. For the record it has nothing to do with "does exactly was Status did before".